Example/Sources/TableViewController.swift

```diff
@@ -52,13 +52,13 @@ final class TableViewController: UITableViewController {
 
         // ** optional editing **
         // if needed, enable the editing functionality on the tableView
-        let tableDataSourceEditingController = TableEditingController(
-            canEditRow: { (tableView, indexPath) -> Bool in
+        let tableDataSourceEditingController: TableEditingController<DataSource<Section<CellViewModel>>> = TableEditingController(
+            canEditRow: { (tableView, indexPath, dataSource: inout DataSource) -> Bool in
                 return true
-        },
-            commitEditing:{ [unowned self] (tableView, editingStyle, indexPath) in
+            },
```

Comment:
The reason that inout didn't work initially is because I had to help the compiler. Initially I had it (tableView, indexPath, dataSource) and not (tableView, indexPath, dataSource: inout DataSource) .😅